### PR TITLE
Fix Gradle plugins

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR provides two major fixes:
 - `assemble-proto.gradle`  plugin error on building after project `clean`;
 - multi-project build with `model-verifier` plugin (failed due to task cyclic dependencies).

Also, in this PR we add `model/README.MD` file explaining the usage of `model-verifier` plugin.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: model/README.MD [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ 
+ compileJava {
+     options.compilerArgs += ["-processor", "io.spine.model.assemble.AssignLookup", "-AspineDirRoot=${rootDir}"]
+ }
+ 
+ dependencies {
+     compileOnly group: 'io.spine.tools', name: 'spine-model-assembler', version: spineVersion
+ }
+ ``` 
+The Spine model elements are `Entity` classes, `.proto` definitions, etc.
```
Comment (armiol) [inline model/README.MD]:
```
The Spine project model is formed by Java and Protobuf declarations of Entities and the messages handled (i.e. `Command`s, `Event`s and `Rejection`s). 
```

File: model/README.MD [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ buildscript {
+     dependencies {
+         classpath "io.spine.tools:spine-model-verifier:${spineVersion}"
+     }
+ }
+ 
+ apply plugin: 'io.spine.tools.spine-model-verifier'
+ 
+ compileJava {
+     options.compilerArgs += ["-processor", "io.spine.model.assemble.AssignLookup", "-AspineDirRoot=${rootDir}"]
```
Comment (armiol) [inline model/README.MD]: `-AspineDirRoot` seems to be a typo. Isn't it `-D` that preceeds the command-line parameters addressed to the application?

File: scripts/assemble-proto.gradle [hunk 3]
```diff
@@ -73,10 +73,28 @@ Collection<File> collectProto() {
...
+                 * the execution, the task should complete successfully.
+                 *
+                 * To make sure the configuration phase passes, we suppress the GradleException
+                 * thrown by `zipTree()` indicating that the given file, which is a dependency JAR
+                 * file does not exist.
+                 *
+                 * As a side effect, the message is shown upon `./gradlew clean build` or upon
+                 * a newly created version of framework build etc.
+                 */
+                logger.error(
```
Comment (armiol) [inline scripts/assemble-proto.gradle]: Let's discuss.

**Pull request merged**